main.nf

```diff
@@ -928,31 +965,35 @@ process spades {
         saveAs: {filename -> filename.indexOf(".fastq.gz") == -1 ? "Assembly/SPAdes/$filename" : null}
 
     input:
-    set id, file(sr) from trimmed_reads_spades  
+    set id, file(sr) from trimmed_reads_spades
 
     output:
     set id, val("SPAdes"), file("${id}_graph.gfa") into assembly_graph_spades
     set val("SPAdes"), val("$id"), file("${id}_scaffolds.fasta") into (assembly_spades_to_quast, assembly_spades_to_metabat)
     file("${id}_contigs.fasta")
     file("${id}_log.txt")
+    val("${task.cpus}") into ch_spades_cpus
```

Comment:
cant find where `ch_spades_cpus` is used. Probably this is supposed to be reported somewhere?

Reply:
ups sorry, this should have been deleted. Thanks!